manifest/net.dfgHiatus/FuckMirrors/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "FuckMirrors",
+	"id": "net.dfgHiatus.FuckMirrors",
+	"description": "Cringe remover. Locally disables world-space cameras. Breaks things such as UI elements that rely on cameras to render textures. Can sometimes solve rendering issues with certain HMDs.",
+	"category": "Memes",
+	"sourceLocation": "https://github.com/dfgHiatus/FuckMirrors",
+	"versions": {
+		"2.0.0": {
```

Comment:
technically version string doesn't match "2.0.0/" as presented in the mod

Reply:
Uploaded mod binary with correct version string